src/afa/afa.cc

```diff
@@ -195,12 +755,13 @@ Mata::Parser::ParsedSection Mata::Afa::serialize(
 	}
 	parsec.dict["Final"] = fin_states;
 
-	for (const auto& trans : aut.transitions) {
+	/*for (const auto& trans : aut.transitions) {
```

Comment:
Please, don't keep commented code in the PR.

Reply:
And if there is some TODO, explain what is actually there to do.

Reply:
Thanks for your suggestion, I have overlooked it. The comment is removed now.